cmake/module/TryAppendCXXFlags.cmake

```diff
@@ -49,7 +43,7 @@ function(try_append_cxx_flags flags)
     TACXXF                            # prefix
     "SKIP_LINK"                       # options
     "TARGET;VAR;SOURCE;RESULT_VAR"    # one_value_keywords
-    "IF_CHECK_PASSED;IF_CHECK_FAILED" # multi_value_keywords
+    "IF_CHECK_PASSED"                 # multi_value_keywords
```

Comment:
nit: I've also removed `IF_CHECK_FAILED` in [`26cbe1e` (#30732)](https://github.com/bitcoin/bitcoin/pull/30732/commits/26cbe1e95d7960c6b4de19b77240b62f4f27d73b), consider readjusting the comment alignment as well